test.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'minitest/autorun'
+require_relative 'parser'
+
+class Parser < Minitest::Test
+  parallelize_me!
+
+  def setup
+    @verbs_list = get_phrasal_verbs(URI_BASE)
+    @sample_verb = 'Throw Out'
+    @sample_page = Nokogiri::HTML(open('https://www.skypeenglishclasses.com/english-phrasal-verbs/throw-out/'))
+  end
+
+  def test_verb_list_containing_390_hashes
```

Comment:
Use `test` instead of `def`, it's more readable 

```suggestion
  test 'verb list containing 390 hashes'
```